### ISSUE TITLE
huge temporary file

### ISSUE THREAD

**Issue by Jibec** [translate/translate#4177]
translate toolkit uses 21,6GB of space on my drive:
```
[jb@localhost .translate_toolkit]$ ls -lah
total 21G
drwxrwxr-x.  2 jb jb 4,0K 23 nov.  19:00 .
drwx--x---+ 83 jb jb  12K 23 nov.  12:37 ..
-rw-r--r--.  1 jb jb  21G 23 nov.  19:00 stats.db
```

is this temporary file really needed in home folder?
maybe storing it in tmp would make it faster and more efficient?

as an example, this also create issues when using translate toolkit in automation: https://pagure.io/fedora-docs/translations-scripts/pull-request/18# 
because of using pocount there: https://pagure.io/fedora-docs/translations-scripts/blob/master/f/stats/compute.sh#_11

**Comment by nijel** [comment]
The pocount scripts you are using is relying on [statsdb](https://github.com/translate/translate/blob/master/translate/storage/statsdb.py) which stores this sqlite database in `~/.translate_toolkit/stats.db`. It seems to cache stats for all files ever counted, but that's what I could figure out from quick look at the code and I've seen it for the first time now.

**Comment by nijel** [comment]
Fixed by removing statsdb in  #4188.
